sys/include/net/sock/dtls.h

```diff
@@ -491,13 +491,33 @@
 extern "C" {
 #endif
 
+/**
+ * @ingroup net_sock_util_conf
```

Comment:
I just realized that this is misplaced. We need a new group for the Sock DTLS configurations. Fell free to squash and push this change directly @akshaim

Reply:
Since it was just one configuration, was not sure if I should create another group. Should I create subgroups for `sock_uti`l and `sock_dtls`?

Reply:
This option is not related to sock_utils, so I'd say it should have its own group (like it has its own entry in Kconfig)

Reply:
Oh okay. But I was suggesting a super group for sock and subgroups for `sock_util` and `sock_dtls` . What do you think ?

Reply:
Ah, yeah that makes sense to me.